fiasko_bro/__init__.py

```diff
@@ -1,3 +1,7 @@
 from .code_validator import CodeValidator, validate_repo
-from .validator_helpers import tokenized_validator
+from .validator_helpers import (
+    tokenized_validator_run_if_any,
+    tokenized_validator_run_if_all,
+    tokenized_validator_run_if
```

Comment:
Create the module `tokenized_validators` and place the methods `run_if_any`, `run_if_all`, `run_if` there. Then import the module here. In the end, the API will look like
```
from fiasko_bro import tokenized_validators

@tokenized_validators.run_if_any(tokens=tokens)
```